<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComparisonSelectionWidget</class>
 <widget class="QFrame" name="ComparisonSelectionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>156</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="widgetLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>5</number>
   </property>
   <item row="3" column="1">
    <widget class="QTableView" name="comparisonSelectionTableView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::AllEditTriggers</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>250</number>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>100</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="addComparison">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Add Threshold Criteria</string>
       </property>
       <property name="statusTip">
        <string/>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/add2.png</normaloff>:/add2.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeComparison">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Delete Threshold Criteria</string>
       </property>
       <property name="statusTip">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/delete2.png</normaloff>:/delete2.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Select Attribute Matrix</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="dataContainerComb2"/>
     </item>
     <item>
      <widget class="QComboBox" name="attributeMatrixCombo2"/>
     </item>
     <item>
      <widget class="QPushButton" name="m_SelectedAttributeMatrixPath">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { /* all types of tool button */
    border: 1px solid #8f8f91;
    border-radius: 4px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #DDDDDD, stop: 1 #FFFFFF);
	font-size: 12pt;
	padding-left: 16px;
	padding-right: 4px;
	padding-top: 2px;
	padding-bottom: 2px;
}

QPushButton::menu-indicator {
   /*  image: url(menu_indicator.png); */
    subcontrol-origin: padding;
    subcontrol-position:  left; /* */
}

QPushButton::menu-indicator:pressed, QPushButton::menu-indicator:open
{
    position: relative;
    /*  top: 2px; left: 2px; shift the arrow by 2 px */
}

QPushButton:pressed
{
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #AAAAFF, stop: 1 #FFFFFF);
}

QPushButton:flat 
{
    border: none; /* no border for a flat push button */
}

QPushButton:hover
{
	border: 1px solid #8f8f91;
	border-radius: 4px;
}
</string>
       </property>
       <property name="text">
        <string notr="true">Default / Path / Array</string>
       </property>
       <property name="iconSize">
        <size>
         <width>2</width>
         <height>2</height>
        </size>
       </property>
       <property name="shortcut">
        <string>Ctrl+S</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">QLabel
{
font: 10pt &quot;Arial&quot;;
font-weight: bold;
}</string>
     </property>
     <property name="text">
      <string>Attribute Arrays to Threshold</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../Resources/icons/images/Icons.qrc"/>
 </resources>
 <connections/>
</ui>
